src/app/ecosystem-explorer/project-form/services/github/index.ts

```diff
@@ -1,120 +1,68 @@
-'use server'
-
-import slugify from 'slugify'
+import { ECOSYSTEM_PROJECTS_DIRECTORY_PATH } from '@/constants/paths'
 
 import { getTodayISO } from '@/utils/dateUtils'
-import { encrypt } from '@/utils/encryption'
+
+import type { FormattedLogo } from '../../types'
 
 import { createBlob } from './api/createBlob'
 import { createCommit } from './api/createCommit'
 import { createPR } from './api/createPr'
 import { createTreeBlobs } from './api/createTreeBlobs'
 import { getLatestCommitOnMain } from './api/getLatestCommitOnMain'
-import type { AllowedImageFormats } from './utils/fileUtils'
-import {
-  getMarkdownTemplate,
-  type MarkdownTemplateParams,
-} from './utils/markdownUtils'
-import { getFolderPaths } from './utils/pathUtils'
 
-type SubmitProjectParams = {
-  name: string
-  email: string
-  timestampISO: string
-  yearJoinedISO: string
+export type SubmitProjectParams = {
+  slug: string
+  markdownTemplate: string
+  prTitle: string
   logo: {
```

Comment:
I think I missed something - Why don't we need to specify allowed image formats anymore?